db/BaseWriteService/index.js

```diff
@@ -0,0 +1,32 @@
+const assert = require("assert");
+const { knex } = require("../helpers");
+const BaseService = require("../BaseService");
+
+class BaseWriteService extends BaseService {
+    insertRecord(object) {
+        assert.ok(typeof object === "object", "object<object> is required");
+
+        return knex.insert(object)
+            .into(this.name)
+            .then((row) => {
+                knex.destroy();
+
+                return row;
+            });
+    }
+
+    deleteRecordById(recordId) {
+        assert.ok(typeof recordId === "number", "recordId<number> is required");
+
+        return knex(this.name)
+            .where("id", recordId)
+            .del()
+            .then((isRemoved) => {
```

Comment:
This is actual number of rows deleted. Better to check that it is === or > depending on fits best.